src/styles/primeng-theme/components/_paginator.scss

```diff
@@ -0,0 +1,20 @@
+.ui-paginator {
+    .ui-paginator-page.ui-state-active {
```

Comment:
Background and border colors needs variables in this file: https://github.com/dotCMS/core-web/pull/350/files#diff-020fe844182b21b4f7ee91ed255735ee